test/scenarios/pollutant/atmo-france/test-tabular-success.bats

```diff
@@ -65,6 +65,9 @@ load "../../../support/test-actions.bash"
 }
 
 @test "Test real atmo tabular server pollutant data" {
+    skip "This test requires access to the real Atmo France server, which is not available currently."
+    # https://tabular-api.data.gouv.fr/api/resources/d2b9e8e6-8b0b-4bb6-9851-b4fa2efc8201/ returns a 404.
+
```

Comment:
The UI to browse the tabular data is broken too: https://explore.data.gouv.fr/fr/datasets/6149925a2ff0ab6cebdd6fe8/#/resources/d2b9e8e6-8b0b-4bb6-9851-b4fa2efc8201